### PR TITLE
Rebuild Simpla

### PR DESCRIPTION
**Pull request by bedeoverend**
Complete rebuild of Simplas JS section, building the underlying core on an events / redux based system.

### CODE REVIEW

**Comment by bedeoverend** [comment]
Seems to still be failing - mostly it seems just due to Polyfills, it's missing at least `Promise` and `assign`

File: demo/index.html [hunk 1]
```diff
@@ -6,13 +6,15 @@
...
       Simpla({
         project: 'NJQrIRWng',
-        base: 'https://elements.simpla.io/',
-        elements: [
-          'simpla-img/simpla-img.html',
-          'simpla-text/simpla-text.html',
-          'simpla-block/simpla-block.html',
-          'sm-admin/sm-admin.html'
-        ]
+        elements: {
```
Comment (seaneking) [inline demo/index.html]: Leave the default element array, no need to specify in demo.

File: demo/state.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    </script>
+  </head>
+  <body>
+    <pre><code id="code"></code></pre>
+    <script>
+      function render() {
+        document.getElementById('code').innerHTML = JSON.stringify(Simpla._store.getState(), null, 2);
+      }
+
+      Simpla._store.subscribe(render);
```
Comment (seaneking) [inline demo/state.html]: Don't expose a private method in a demo

**Comment by seaneking** [comment]
also kill simpla element bower deps

**Pull request merged**